supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -359,6 +359,50 @@ There is no functional difference between the first server that was installed an
 
 *See also*:
 
+[[inference]]
+==== image:images/yes.png[yes] inference (noun)
+*Description*: Inference is when a model can process, deduct, and produce outputs based on input data. The process of using a trained AI model to generate predictions or conclusions based on the input data provided to the model.
+
+*Use it*: yes
+
+[.vale-ignore]
+*Incorrect forms*: 
+
+*See also*:
+
+[[`InferenceService`]]
+==== image:images/yes.png[yes] `InferenceService` (noun)
+*Description*: In Red Hat OpenShift AI, this is the Custom Resource  Definition (CRD) used to create the `InferenceService` object. Written as the CRD name `InferenceService` with backticks. 
+
+*Use it*: yes
+
+[.vale-ignore]
+*Incorrect forms*: InferenceService, inference serving, inferenceservice 
```

Comment:
Was this incorrect form supposed to be "inference service"?

(Also going back to my question about whether "inference service" could ever been used generically. It's fine if it's how you want to do it, but I also express my concerns about references to a certain concept always having to be in pascal case/monospace, since that's not a great reader experience)

```suggestion
*Incorrect forms*: InferenceService, inference serving, inference service 
```

Reply:
I don't think we want to suggest 'inference service' on its own. I only see it in the context of the object, but ill ask the RHOAI folks as well.